project/Dependencies.scala

```diff
@@ -13,23 +13,30 @@ object Dependencies {
   lazy val scalaTestVersion = settingKey[String]("The version of ScalaTest to use.")
   lazy val scalaCheckVersion = settingKey[String]("The version of ScalaCheck to use.")
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
+  lazy val sslConfigVersion = settingKey[String]("The version of ssl-config to use.")
   val junitVersion = "4.12"
-  val sslConfigVersion = "0.3.7"
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
   val aeronVersion = "1.15.1"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.12.8", "2.13.0-M5"),
+    crossScalaVersions := Seq("2.12.8", "2.13.0-RC2"),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
     scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.14.0"),
-    scalaTestVersion := "3.0.7",
+    scalaTestVersion := "3.0.8-RC4",
     java8CompatVersion := {
       CrossVersion.partialVersion(scalaVersion.value) match {
+        // java8-compat is only used in a couple of places for 2.13,
+        // it is probably possible to remove the dependency if needed.
         case Some((2, n)) if n >= 13 => "0.9.0"
         case _                       => "0.8.0"
       }
-    })
+    },
+    sslConfigVersion := {
+      CrossVersion.partialVersion(scalaVersion.value) match {
```

Comment:
this one not bin compat?

Reply:
I was reluctant to make that assumption, though I'm not sure how big that issue is after https://github.com/lightbend/ssl-config/pull/160/files . AFAIK the parts of the API that we're using are compatible, but it seemed nice to avoid changing the transitive dependencies for Scala 2.12 users

Reply:
Run MiMa in ssl-config, comparing against 0.3.7, and review the output.